projects/Mallard/src/services/errors.ts

```diff
@@ -67,6 +69,10 @@ class ErrorService {
         this.hasConsent = hasConsent
         if (this.hasConsent && !this.hasConfigured) {
             this.sentryImpl.config(SENTRY_DSN_URL).install()
+            this.sentryImpl.setTagsContext({
+                environment: isInBeta ? 'BETA' : 'RELEASE',
```

Comment:
`isInBeta` is a function from memory so this will always evaluate to true.

Reply:
Good spot!

Reply:
Fixed.